app/validators/validator_util.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class ValidatorUtil
```

Comment:
How about we make this a parent class, or mixin for the other validators, and name it something more specific (something like `ProgramBaseValidator`)?
```
class ProgramBaseValidator < ActiveModel::Validator
...
class EduProgramValidator < ProgramBaseValidator
...
class ProgramValidator < ProgramBaseValidator
```
Then you can turn the methods into private instance methods, and call them from the subclasses.

Reply:
renamed to BaseValidator since other csvs in future might need a facility_code validation